Don't "link" with uuid library on Mac OS X.

There's no strict uuid library requiring linking flags on
Mac OS X.
=========================================================
--- configure.orig	2026-08-18 15:17:05.000000000 -0400
+++ configure	2026-09-01 16:00:22.601498445 -0400
@@ -13462,7 +13462,6 @@
          UUID_CFLAGS=""
          UUID_LDFLAGS="-L$dir/lib"
          UUID_INCLUDE="-I$MYUUID/include/uuid -I$MYUUID/include"
-         UUID_LIBS="-luuid"
          { $as_echo "$as_me:${as_lineno-$LINENO}: checking for UUIDLIB version" >&5
 $as_echo_n "checking for UUIDLIB version... " >&6; }
          CPPFLAGS="$U_INCLUDE $CPPFLAGS"
@@ -13481,7 +13480,6 @@
     UUID_CFLAGS=""
     UUID_LDFLAGS="-L$MYUUID/lib"
     UUID_INCLUDE="-I$MYUUID/include/uuid/uuid.h -I$MYUUID/include"
-    UUID_LIBS="-luuid"
     { $as_echo "$as_me:${as_lineno-$LINENO}: checking for UUID version" >&5
 $as_echo_n "checking for UUID version... " >&6; }
     CPPFLAGS="$U_INCLUDE $CPPFLAGS"
